.github/workflows/continuous-integration-workflow.yml

```diff
@@ -28,13 +28,13 @@ jobs:
       - name: Install Poetry for Linux
         if: runner.os == 'Linux'
         run: |
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python - --preview --version 1.1.3
+          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python - --preview --version 1.1.4
```

Comment:
why change this version?

Reply:
The poetry is unstable to update packages, so CI tasks failed. I'm trying latest version, see if it's fixed.